lab_1_classify_by_unigrams/main.py

```diff
@@ -106,6 +182,28 @@ def detect_language(
     In case of corrupt input arguments, None is returned
     """
 
+    if (not isinstance(unknown_profile, dict)
+            or not isinstance(profile_1, dict)
+            or not isinstance(profile_2, dict)):
+        return None
+
+    compare_to1 = compare_profiles(unknown_profile, profile_1)
+    compare_to2 = compare_profiles(unknown_profile, profile_2)
+    result = ""
+
+    if compare_to1 is None or compare_to2 is None:
+        return None
+
+    if compare_to1 < compare_to2:
+        result = profile_1["name"]
+    if compare_to2 < compare_to1:
+        result = profile_2["name"]
```

Comment:
and here?

Reply:
same here, that's why I added the variable "result"